webapp/app/components/field/lines.js

```diff
@@ -0,0 +1,76 @@
+import { Container, Graphics } from 'pixi.js';
+
+
+const FIELD_MARGIN_X = 200;
+const FIELD_MARGIN_Y = 50;
+
+export class Lines {
+  constructor({ width, height }) {
+    this.width = width;
+    this.height = height;
+
+    this.stage = new Container();
+    this.graph = null;
+
+    this.createStage();
+    this.handleResize();
+  }
+
+  createStage() {
+    this.graph = new Graphics();
+
+    this.stage.addChild(this.graph);
+
+    this.graph.lineStyle(1, 0xffffff);
+
+    this.graph
```

Comment:
mozesz wrzucic lineStyle przed moveTo

Reply:
chyba ze celowo rozbiles te deklaracje aby rozroznic odpowiednie czesci

Reply:
rowniez te ponizej

Reply:
celowo to rozbiłem, żeby było widać, że ten styl dotyczy całej grupy elementów i później robiłem na poszczególne figury żeby było czytelniej

Reply:
ok, tak myslalem